code/solver.py

```diff
@@ -392,12 +369,12 @@ def train_multi(self):
                 label_trg = label_org[rand_idx]
 
                 if dataset == 'CelebA':
-                    c_org = label_org.clone()
-                    c_trg = label_trg.clone()
-                    zero = torch.zeros(x_real.size(0), self.c2_dim)
-                    mask = self.label2onehot(torch.zeros(x_real.size(0)), 2)
-                    c_org = torch.cat([c_org, zero, mask], dim=1)
-                    c_trg = torch.cat([c_trg, zero, mask], dim=1)
+                    c_org = self.label2onehot(label_org, self.c2_dim)
+                    c_trg = self.label2onehot(label_trg, self.c2_dim)
+                    zero = torch.zeros(x_real.size(0), self.c_dim)
+                    mask = self.label2onehot(torch.ones(x_real.size(0)), 2)
+                    c_org = torch.cat([zero, c_org, mask], dim=1)
```

Comment:
This should be torch.cat([c_org, zero, mask], dim=1), and even the next line

Reply:
Oh you are right; I missed this. And self.c2_dim should be self.c_dim.